app/server.js

```diff
@@ -11,7 +11,7 @@ const config = require('./config/config');
 const localesMap = [
     { lang: 'en', label: 'English' },
     { lang: 'he', label: 'עברית' },
-    { lang: 'sv', label: 'Swedish' },
+    { lang: 'sv', label: 'svenska' },
```

Comment:
Shouldn't it be with capital S? Svenska?

Reply:
nope :) In Swedish, language names start with lowercase letters

Reply:
Learned something new :)